src/main/java/com/tenx/ms/retail/order/rest/dto/OrderStatusEnum.java

```diff
@@ -0,0 +1,35 @@
+package com.tenx.ms.retail.order.rest.dto;
+
+import com.tenx.ms.commons.rest.BaseValueNameEnum;
+import io.swagger.annotations.ApiModel;
+
+@ApiModel(value = "Order Status")
+public enum OrderStatusEnum implements BaseValueNameEnum<OrderStatusEnum> {
+    ORDERED(1, "ORDERED"),
+    PACKING(2, "PACKING"),
+    SHIPPED(3, "SHIPPED"),
+    INVALID(4, "INVALID");
+
+    private int value;
```

Comment:
@marielbysoares on the entity, I used
```java
@Enumerated(EnumType.STRING)
@Column
private OrderStatusEnum status;
```
And here I only wrote:
```java
public enum OrderStatusEnum {
    ORDERED,
    PACKING,
    SHIPPED
}
```

With none of the boilerplate for going from int to string.